src/txmempool.cpp

```diff
@@ -406,7 +406,7 @@ void CTxMemPool::removeUnchecked(txiter it, MemPoolRemovalReason reason)
 {
     CTransactionRef ptx = it->GetSharedTx();
     NotifyEntryRemoved(ptx, reason);
-    if (reason != MemPoolRemovalReason::BLOCK && reason != MemPoolRemovalReason::CONFLICT) {
+    if (reason != MemPoolRemovalReason::BLOCK) {
```

Comment:
In commit "[wallet] Notify conflicted transactions in TransactionRemovedFromMempool" (3529e0bc2cba7a9d502de2c0fe9b17ec94a7fd0c)

Could you add a comment about why notifications are skipped for BLOCK? I believe it's just for performance because BlockConnected notification provides the same information.

Reply:
done